packages/layout/src/ChartPanel.ts

```diff
@@ -121,6 +122,12 @@ export class ChartPanel extends Border2 implements IHighlight {
         return this;
     }
 
+    highlight(_?: boolean) {
```

Comment:
This is the "normal" set/get pattern (it provides better auto completion in the IDE):
```
highlight(): boolean;
highlight(_: boolean): this {
highlight(_?: boolean): boolean | this {
}
```